<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiDigitalSignatures</class>
 <widget class="QWidget" name="UiDigitalSignatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DigitalSignatures</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_Signature">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabName_ED25519">
      <attribute name="title">
       <string>ED25519</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0" colspan="5">
        <widget class="QTextEdit" name="ed25529_textEdit_Message">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="ed25519_pushButton_CalculateSignature">
         <property name="text">
          <string>Calculate signature</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>* If private key already set, it will calculate public key for that.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>32 bytes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Message (text to be signed or verified)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QLineEdit" name="ed25529_textEdit_privateKey">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ed25519_label_PublicKey">
         <property name="text">
          <string>Public key</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Private key size</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QLineEdit" name="ed25529_textEdit_publicKey">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_Signature_ED25519_Clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Signature</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="ed25519_pushButton_VerifySignature">
         <property name="text">
          <string>Verify signature</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="ed25519_pushButton_GenerateKeysPair">
         <property name="text">
          <string>Generate keys pair</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="5">
        <widget class="QLineEdit" name="ed25529_textEdit_Signature">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ed25519_label_PrivateKey">
         <property name="text">
          <string notr="true">Private key</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabName_Signature_RSA">
      <attribute name="title">
       <string>RSA</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
